index.js

```diff
@@ -0,0 +1,108 @@
+// import Book from modules
+import addToLocalStorage from './modules/localStorage.js';
+import Book from './modules/shelfClass.js';
+
+let books;
+
+const book1 = new Book('title1', 'author1', '001');
+const book2 = new Book('title2', 'author2', '002');
+const book3 = new Book('title3', 'author3', '003');
+
+books = [book1, book2, book3];
+
+// add event listner to newly added book remove button
+let removeBtn = document.querySelectorAll('.remove');
+
+function removeBook(ref) {
+  const result = books.filter((value) => value.id !== ref);
+  books = result;
+  addToLocalStorage(books);
+}
+
+function addEvents() {
+  removeBtn.forEach((element) => {
+    element.addEventListener('click', (e) => {
+      const ref = e.target.id;
+      removeBook(ref);
+      e.target.parentElement.remove();
+    });
+  });
+}
+
+// populate dom with the list
```

Comment:
- You can make your code cleaner by removing all comments. Your code is talking about everything itself 👍